manifests/server.pp

```diff
@@ -70,11 +70,13 @@
 ) inherits puppetdb::params {
   # deprecation warnings
   if $database_ssl != undef {
-    warning('$database_ssl is deprecated and will be removed in the next major release. Please use $jdbc_ssl_properties = "?ssl=true" instead.')
+    warning('$database_ssl is deprecated and will be removed in the next major release.
```

Comment:
This mechanism, where you are breaking up the lines as such to achieve < 80 chars is going to mess with the output formatting of the warning.

See this for an example: https://gist.github.com/kbarber/1fb25ad0b2b70383fb3867a25b7fd451